<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>583</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Vulkan Configurator</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>281</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Profiles</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="2">
      <widget class="QListWidget" name="listWidgetProfiles">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBoxAppList">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Apply to specific apps</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <widget class="QPushButton" name="pushButtonAppList">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Edit List...</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonActivateProfile">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>120</y>
      <width>151</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Set as Active Profile</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonEditProfile">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>150</y>
      <width>151</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit Profile...</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>441</width>
      <height>221</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideMiddle</enum>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Break Points</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Message Filters</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>493</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Action</string>
    </property>
    <addaction name="actionHistory"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionVulkan_Info"/>
    <addaction name="actionVulkan_Installation"/>
    <addaction name="actionCustom_Layer_Paths"/>
    <addaction name="separator"/>
    <addaction name="actionLog_API_Dump"/>
    <addaction name="actionSet_Test_Application"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionHistory">
   <property name="text">
    <string>History...</string>
   </property>
  </action>
  <action name="actionSave_Configuration_2">
   <property name="text">
    <string>Save Profile...</string>
   </property>
  </action>
  <action name="actionExit_and_Keep">
   <property name="text">
    <string>Exit and Keep</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionVulkan_Info">
   <property name="text">
    <string>Vulkan Info</string>
   </property>
  </action>
  <action name="actionVulkan_Installation">
   <property name="text">
    <string>Vulkan Installation Analysis</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About VkConfig</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>VkConfig Help</string>
   </property>
  </action>
  <action name="actionSet_Test_Application">
   <property name="text">
    <string>Launch &amp;&amp; Monitor Application...</string>
   </property>
  </action>
  <action name="actionCustom_Layer_Paths">
   <property name="text">
    <string>Custom Layer Paths...</string>
   </property>
  </action>
  <action name="actionLog_API_Dump">
   <property name="text">
    <string>Log API Dump...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
